pkg/apps/application-list.jsx

```diff
@@ -99,6 +99,10 @@ class ApplicationRow extends React.Component {
     }
 }
 
+// We try to install these packages, but only if they are available.
+//
+const appstream_collection_packages = [ "appstream-data" ];
```

Comment:
That is the Fedora name. In Debian, the DEP-11 data is part of apt's metadata with the usual `apt-get update`. But like on Fedora this needs to be explicitly enabled, to avoid imposing the overhead on all machines that don't need/want it. Installing the `appstream` package ships an apt config snippet which will cause the appstream metadata to get downloaded.

However, in Fedora the "appstream" package also exists (for the tools). I actually liked your original proposal for making that a configure option. However, even with that it wouldn't work perfectly yet as one would need to run apt-get update (or pkcon refresh) *after* appstream gets installed (and here it's installed at the beginning).

It's probably fine to leave this broken on Debians for now, but just wanted to mention that this isn't quite an universal approach. I don't know what Gentoo etc. are doing.

Reply:
> Installing the appstream package ships an apt config snippet which will cause the appstream metadata to get downloaded.

Great, I didn't know this, but always wanted to find out. :)

Reply:
> It's probably fine to leave this broken on Debians for now, but just wanted to mention that this isn't quite an universal approach.

I'll (try to) open another PR that unbreaks the configure time creation of manifests, which can then use to make this work for Debian etc.